atc/django-atc-demo-ui/atc_demo_ui/templates/atc_demo_ui/base.html

```diff
@@ -10,6 +10,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
     <!-- React.js -->
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/2.2.0/es5-shim.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/2.2.0/es5-sham.min.js"></script>
```

Comment:
sham and shim each of them cover different functions: https://github.com/es-shims/es5-shim

Sham was not needed until react 0.13 (from my experiment)